lib/translations.json

```diff
@@ -303,8 +315,132 @@
       "value": "Diskussionsbeiträge"
     },
     {
-      "key": "testimonial/sequenceNumber",
-      "value": "Abo #{num}"
+      "key": "testimonial/meta/pageTitle",
```

Comment:
FYI, feel free to manually remove strings that I'm working on after running the translations script (just in case you only your changes to appear here).